src/radix_connect/well_known_client/models/dapp_definitions.rs

```diff
@@ -0,0 +1,55 @@
+use crate::prelude::*;
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct DappDefinitions {
```

Comment:
`DappDefinitions` lacks documentation!

It also lacks a constructor! Write a constructor which uses `impl ItoIterator<Item = DappDefinition>` and `impl Into<Option<RCMCallbackPath>>`